k8s.gcr.io/images/k8s-staging-networking/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
Humm, may need to include the boilerplate?

Reply:
The parsing error was because I accidentally had a tab in there...
I added the boilerplate, although every sub-project here seems to have a different boilerplate message...

Reply:
Haha, thanks for finding it out :)